lib/autoupdate/tools/helper.go

```diff
@@ -27,16 +27,13 @@ import (
 	"github.com/gravitational/trace"
 
 	"github.com/gravitational/teleport"
-	"github.com/gravitational/teleport/lib/autoupdate"
 	stacksignal "github.com/gravitational/teleport/lib/utils/signal"
 )
 
 // Variables might to be overridden during compilation time for integration tests.
 var (
 	// version is the current version of the Teleport.
 	version = teleport.Version
-	// baseURL is CDN URL for downloading official Teleport packages.
-	baseURL = autoupdate.DefaultBaseURL
```

Comment:
this `baseURL` is overridden by integration test, same as `version` variable
https://github.com/gravitational/teleport/blob/f93604062279489c2e880e9456cbe532b768459c/integration/autoupdate/tools/main_test.go#L163-L177